content/tutorial/react/index.md

```diff
@@ -142,53 +204,23 @@ Navigate your Register or Login links or buttons to the following URLs:
 
   ```JavaScript
   import React from "react"
-  import { withRouter } from "react-router-dom";
-
-  const apiKey = '{{ Your API Key }}';
-
-  class Login extends React.Component {
-    constructor(props) {
-      super(props);
-      this.state = {
-        userProfileResponse: null
-      }
-    }
-
-    componentDidMount() {
-      const token = new URLSearchParams(this.props.location.search).get("token");
-      fetch("https://api.loginradius.com/identity/v2/auth/account?apikey=" + apiKey, {
-        method: 'GET',
-        headers: {
-          'Authorization': 'Bearer ' + token
-        },
-      })
-        .then(res => res.json())
-        .then(res => {
-          this.setState({ userProfileResponse: res })
-        })
-        .catch(e => {
-          console.log(e);
-        })
-    }
-
-    render() {
-      const { userProfileResponse } = this.state;
-
-      return (
-        <div>
-          <span style={{ whiteSpace: "pre-wrap", textAlign: "left" }}>
-            {JSON.stringify(userProfileResponse, null, 4)}
-          </span>
-        </div>
-      );
-    }
-  }
-
-  export default withRouter(Login);
+  import { useLRAuth, withAuthenticationRequired } from "loginradius-react";
+  
+  const Login = () => {
+  const { user } = useLRAuth();
+    return (
+    <div>
+      <span style={{ whiteSpace: "pre-wrap", textAlign: "left" }}>
+        {JSON.stringify(user, null, 4)}
+      </span>
+    </div>
+     );
+  } 
+  export default withAuthenticationRequired(Login, {
+  onRedirecting: () => <Loading />, returnTo: '/login',
+  });
```

Comment:
Here the component name should be **Profile** i think?

Reply:
Component is named Login for consistency.